### DISCUSSION TITLE
Is there a way of adding some sort of tagging in the dependency dashboard to the PRs?

### DISCUSSION THREAD

**Discussion by pbabbicola** [renovatebot/renovate#32397] [Request Help] [answered]
### How are you running Renovate?

A Mend.io-hosted app

### Please tell us more about your question or problem

Situation:
Where I work, we have a monorepo that includes several services and languages. The part I'm most concerned about, and I'm often responsible about, is the backend dependencies.
We have a lot of Go services which more often than not have the same dependencies. These services aren't necessarily maintained by the same teams.
For example, we have a `service A` that has `dependency 1` and a `service B` that also has `dependency 1`. These packages are normally updated in one PR created by Renovate. These packages then tag the reviewers from the corresponding teams via our Renovate configuration. So far, so good.
For many reasons (that are probably irrelevant), I would like to move to a model in which we approve the dependencies before creating the PRs. Is there any way we could somehow tag these updates, in a way that is mergeable, and that can be seen from the Dependency Dashboards? Renovate "knows" already which service corresponds which each set of reviewers, but I haven't found any way in which I can add either some tags, mergeable prefixes or suffixes to the PR names, or similar.
Thanks!

**Comment by rarkins** [comment]
I think the most obvious solution would be:
- You assign `labels` via Renovate config if not already, and
- The dashboard displays the labels next to the branch name

**Reply by pbabbicola** [reply]
Does it? I haven't found any documentation that said this. I will try it out!

**Reply by rarkins** [reply, marked as answer]
Sorry, I meant that as a proposed solution. Although it's been talked about in the past, I don't think we ever completed the second point. Please "Suggest an idea" in a new discussion with a mockup of the markdown you'd hope to see if not

**Reply by pbabbicola** [reply]
Thank you!
